doc-content/enterprise-only/cases/case-management-case-file.con.adoc

```diff
@@ -22,3 +22,53 @@ You can specify any prefix. For example, if you specify the prefix `IT`, the fol
 `IT-0000000002`
 
 `IT-0000000003`
+
+You can also configure a variable as the business key, for example the following generates the `HR-TYPE1` prefix.
+
+[source]
+----
+{
+  case variable =>  type = "type1"
+  prefix = "HR-@{UPPER(type})"
+}
+----
+
+.Prefix HR-TYPE1
+
+[source]
+----
+{
+  HR-TYPE1-0000000001
+  HR-TYPE1-0000000002
+  HR-TYPE1-0000000003
+}
+----
+
+.Adding a different case variable, for example, ``type = 3type`
```

Comment:
Is this a subheading? If so I would do this:
.Adding a different case variable
The following example shows a sequence where the `type` variable has the value `3type1. 
Something like that. I still don't see this in rendered output so be sure to check.

Reply:
Done in Design doc. This topic removed for GSG.